src/p2p/net_node.inl

```diff
@@ -1788,6 +1808,35 @@ namespace nodetool
   }
   //-----------------------------------------------------------------------------------
   template<class t_payload_net_handler>
+  bool node_server<t_payload_net_handler>::extend_make_next_connection_after(epee::net_utils::zone zone,
+                                                                             bool acquire_peers) {
+    if (zone == epee::net_utils::zone::public_ || !m_exclusive_peers.empty())
+      return false;
+
+    auto scaling_factor = 1;
+    if (acquire_peers)
+      scaling_factor = ::config::ACQUIRE_PEERS_RECONNECT_SCALING_FACTOR;
```

Comment:
I would recommend using [`std::chrono::system_clock`](https://en.cppreference.com/w/cpp/chrono/system_clock) with `std::chrono::milliseconds`, etc. throughout this section.

Reply:
Will change.